extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+lines = sensors_data.readlines()
+
+
+def sensors(lines):
+    #global lines
+    candidate_lines = []
+    for line in lines:
+        line = line.decode()
+        if line.startswith('kat.sensors'):
+            for candidate_line in line.split():
+                candidate_lines.append(candidate_line)
+    return candidate_lines
+
+# Loop over candidate_lines and extract sensor names
+def sensor_names():
+    sensors_list = []
+    candidate_lines = sensors(lines)
```

Comment:
What happens if you decide to rename `def sensors`, it would simply mean that you have to rename here as well. Perhaps, parse sensors as an argument.

eg: 
``` 
    def name_of_func(sensor_list):
       candidate_lines = sensor_list
       ....
    sensor_list = name_of_func_that_gets_sensor_list(output_from_URL)
    var_something = name_of_func(sensor_list)
```